.github/workflows/python-publish.yml

```diff
@@ -40,13 +40,12 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.10.x"
-      - name: Install dependencies
+      - name: Install pip, build and twine
         run: |
-          python -m pip install --upgrade pip
-          pip install setuptools setuptools-scm wheel twine
-      - name: Build a binary wheel and a source tarball
-        run: |
-          python setup.py sdist bdist_wheel
+          python -m pip install --upgrade --disable-pip-version-check pip
+          python -m pip install --upgrade build twine
```

Comment:
ich würde lieber auch die build-dependencies in einem requirements-file haben, so dass sie auch gepinnt sind.
schau dir mal das template-repo an der stelle an:
https://github.com/Hochfrequenz/python_template_repository/blob/b7eed0cb58ecc0c4722159111bcbd621a0c0698c/.github/workflows/python-publish.yml#L48

keine überraschungen mehr. alles gepinnt und reproduzierbar.

Reply:
fair point. Fixed in b7177e07ea914c07cc675dd8ee08151f9fed0c5e und 088cc5d7b00994f7df4b0d86f9c8ac77f7930b7d